fractions/keycloak-microprofile-jwt/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2015 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.thorntail</groupId>
+    <artifactId>build-parent</artifactId>
+    <version>2.3.1.Final-SNAPSHOT</version>
+    <relativePath>../../build-parent/pom.xml</relativePath>
+  </parent>
+
+  <groupId>io.thorntail</groupId>
+  <artifactId>keycloak-microprofile-jwt</artifactId>
+
+  <name>Keycloak Microprofile JWT</name>
+  <description>Securing RESTful services with Keycloak and MP JWT</description>
+
+  <packaging>jar</packaging>
+
+  <properties>
+    <swarm.fraction.stability>stable</swarm.fraction.stability>
+    <swarm.fraction.tags>Security,JWT,Web</swarm.fraction.tags>
+  </properties>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.thorntail</groupId>
+      <artifactId>microprofile-jwt</artifactId>
+    </dependency>
+    
+    <!-- Provided APIs -->
+    <dependency>
+      <groupId>org.keycloak</groupId>
+      <artifactId>keycloak-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.keycloak</groupId>
+      <artifactId>keycloak-adapter-core</artifactId>
+      <version>${version.keycloak}</version>
+    </dependency>
```

Comment:
Are these really meant to be exposed to the user? I removed the `keycloak-admin-client`, as that looked really weird, but could add that back if that was intentional.

Reply:
I only added this one to deal with some of the missing HTTP client API classes :-), so sure it can go away, thanks

Reply:
But the other two are correct? I mean, does depending on this fraction really mean that some Keycloak APIs are now available to the user? Or is that only for the fraction itself (in which case these deps should be `<scope>provided</scope>`)?

Reply:
I've added the provided scope, as indeed Keycloak API should not be exposed (users can use 'keycloak' fraction if they need Keycloak API) thanks